game_event/views.py

```diff
@@ -150,3 +154,38 @@ def process_answer_game_event(request, id):
     content['event_exists'] = True
 
     return render(request, 'game_event/result.html', content)
+
+
+def get_weather_from_api(game_events, weather_getter):
+    weather = []
+    for game in game_events:
+        url = 'https://api.openweathermap.org/data/2.5/weather'
+        params = {
+            'q': game.court.city,
+            'units': 'metric',
+            'appid': os.getenv("APPID")
```

Comment:
@SchSeba in case it's empty, it will just fail fetching the weather (there's a test case for that). This is also what allows running the app without providing this env var (i.e. it will work as usual but will not populate weather details).

I don't think there's an added value with checking the env var.

Reply:
@OhadSeg we should let @SchSeba resolve his own comment.

Reply:
ok my bad @SchSeba @yftacherzog

Reply:
works for me